src/components/themed/WalletListSwipeRow.js

```diff
@@ -70,6 +70,7 @@ class WalletListSwipeRowComponent extends React.PureComponent<Props & ThemeProps
 
     this.closeRow()
     this.props.selectWallet(walletId, currencyCode)
+    Keyboard.dismiss()
```

Comment:
Why not just put the "Keyboard.dismiss()" inside the "this.closeRow" function instead of calling it multiple times?

Reply:
Whoops, good catch. Fixed